sdk/eventhub/event-hubs/src/eventProcessor.ts

```diff
@@ -216,7 +219,20 @@ export class EventProcessor {
             partitionId: partitionId
           };
 
-          const checkpointManager = new CheckpointManager();
+          const partitionOwnership: PartitionOwnership = {
+            eventHubName: this._eventHubClient.eventHubName,
+            consumerGroupName: this._consumerGroupName,
+            instanceId: generate_uuid(),
```

Comment:
This should be `this._id`, since the `instanceId` is meant to match the id of the `EventProcessor`